dimod/generators/bpsp.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2024 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import dimod
+import numpy.random as rnd
```

Comment:
```suggestion
import numpy.random as rnd

import dimod
```
this is the height of nitpicking, but https://peps.python.org/pep-0008/#imports, specifically
```

    Standard library imports.
    Related third party imports.
    Local application/library specific imports.

```

Reply:
I would also just do `import numpy.random` and then use the fully qualified import below. Rather than introduce `rnd` which when someone sees later they may not recognize.